tests/python/unittest/test_io.py

```diff
@@ -255,7 +257,7 @@ def check_libSVMIter_news_data():
 
     check_libSVMIter_synthetic()
     check_libSVMIter_news_data()
-
+@unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/7626")
```

Comment:
nit: Could you put the right link to the issue?

Reply:
I would say that is more than a nit :)

Reply:
nit here: please add a new line this line

Reply:
Took care of the comment.